src/d/actor/d_a_swhit0.cpp

```diff
@@ -3,341 +3,491 @@
 // Translation Unit: d_a_swhit0.cpp
 //
 
-#include "d_a_swhit0.h"
-#include "dolphin/types.h"
+#include "d/actor/d_a_swhit0.h"
+#include "d/d_com_inf_game.h"
+#include "d/d_procname.h"
+#include "f_op/f_op_actor_mng.h"
+#include "JSystem/JKernel/JKRHeap.h"
+#include "JSystem/JUtility/JUTAssert.h"
+#include "m_Do/m_Do_mtx.h"
+
+static dCcD_SrcSph l_sph_src = {
+    // dCcD_SrcGObjInf
+    {
+        /* Flags             */ 0,
+        /* SrcObjAt Type     */ 0,
+        /* SrcObjAt Atp      */ 0,
+        /* SrcObjAt SPrm     */ 0,
+        /* SrcObjTg Type     */ 0xFF1DFEFF,
```

Comment:
I would write this with enums like `~(AT_TYPE_WATER | AT_TYPE...)` so it's more readable

Reply:
Done!